README.md

```diff
@@ -5,7 +5,8 @@ A HTML5 fullscreen plugin for Leaflet.
 
 ``` js
 var map = new L.Map('map', {
-    fullscreenControl: true
+    fullscreenControl: true,
+    fullscreenPseudo: false // if true, fullscreen to page width and height
```

Comment:
If you're implementing a custom button _and_ doing pseudo-fullscreen rather than HTML5 I think you're better off not using this plugin at all -- at that point you're bypassing 95% of its functionality.